_posts/2023-11-03-java-21.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Trino is moving to Java 21"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/images/logos/java-duke-21.png
+show_pagenav: false
+---
+
+We're excited to announce that as of version 432, Trino is running with Java 21
```

Comment:
We may want to clarify that this is for the Trino Docker image

Reply:
How about the updated wording?